samples/subapp2-store/src/subapps/main-body.tsx

```diff
@@ -52,7 +52,7 @@ export const subapp: ReactSubApp = {
     reduxFeature({
```

Comment:
what are these changes in this file for?

Reply:
I removed the subapp-web which was not required here - I used it when I was playing with that.

Reply:
what does these changes have to do with subapp-web?